### DISCUSSION TITLE
Advanced Stage Selection Options

### DISCUSSION THREAD

**Discussion by troy99culo** [ikemen-engine/Ikemen-GO#2093] [Feature Request]
### Is your feature request related to a problem? Please describe.

What people do nowadays for a stage select screen that looks like its own different screen, is the stage portrait trick. Basically they bake-in everything to stage portraits so theyre fullscreen sprites. Honestly its a clever trick but it'd be better if we didn't need to use workarounds like this.

### Describe the solution you'd like

A feature that'd help a lot is optional stuff like Stage Select before character select, just like in CVS2. Or after select screen with fade in or custom anims like USFIV4. Column, row or alt slot options for stages would be a great option to have as well.

### Search terms

stage select, stage fade

**Comment by jenngra505** [comment]
TMNTxJL used source code modification to have a more advanced stage select screen than currently possible. One thing to note is that the game's system.def file contained this parameter.
stagetransitionscene=data/stageselect.def

**Comment by potsmugen** [comment]
A stage select screen after the character select one doesn't sound too hard to add since all the building blocks are already there. It'd be taking the stage select in the select screen and making it its own screen.

Like a lot of things the challenge will be accounting for old screenpacks as well as new ones. I guess it'd be kind of like the continue screen. With a screenpack parameter to pick either the old method or the new one.
